ironfish/src/rpc/routes/accounts/getStatus.test.ts

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import { v4 as uuid } from 'uuid'
+import { createRouteTest } from '../../../testUtilities/routeTest'
+import { Account } from '../../../wallet'
+
+describe('Route account/status', () => {
+  const routeTest = createRouteTest(true)
+  let account = {} as Account
+
+  beforeAll(async () => {
+    account = await routeTest.node.wallet.createAccount(uuid())
+    await routeTest.node.wallet.setDefaultAccount(account.name)
```

Comment:
You don't need to call this, you can pass the second parameter to createAccount as true.

Also I don't feel strongly but since we have 1 test I would just move this into the test